os/arch/arm/src/s5j/s5j_rtc_lowerhalf.c

```diff
@@ -290,29 +291,29 @@ static struct s5j_lowerhalf_s g_rtc_lowerhalf = {
  * Public Functions
  ****************************************************************************/
 /****************************************************************************
- * Name: s5j_rtc_lowerhalf
+ * Name: s5j_rtc_initialize
  *
  * Description:
- *   Instantiate the RTC lower half driver for the S5J.
- *   General usage:
- *
- *     #include <tinyara/rtc.h>
- *     #include "s5j_rtc.h"
- *
- *     struct rtc_lowerhalf_s *lower;
- *     lower = s5j_rtc_lowerhalf();
- *     rtc_initialize(0, lower);
+ *   Register the RTC driver to provide RTC functionality
  *
  * Input Parameters:
  *   None
  *
  * Returned Value:
- *   On success, a non-NULL RTC lower interface is returned. NULL is
- *   returned on any failure.
+ *   None
  *
  ****************************************************************************/
-FAR struct rtc_lowerhalf_s *s5j_rtc_lowerhalf(void)
+
+void s5j_rtc_initialize(void)
```

Comment:
Currently, rtc_lowerhalf_s type is not useful but generally it had better keep lowerhalf type, then it is easy to add multiple driver in future. I don't think multiple RTC is possible for us, but it is better option..